tools/sync.js

```diff
@@ -75,21 +75,20 @@ const normalizeTX = async (txData, receipt, blockData) => {
     timestamp: blockData.timestamp,
   };
 
-  if (receipt.status) tx.status = receipt.status;
+  if (receipt.status) {
+    tx.status = receipt.status;
+  }
 
   if (txData.to) {
     tx.to = txData.to.toLowerCase();
     return tx;
-  }
-  if (tx.creates) {
+  } else if (tx.creates) {
```

Comment:
@btenterprise2020 can this ever be true? the tx is defined above and it doesn't have `creates` defined, am I missing something?

Reply:
Good catch, it should be txData.creates field that is supported by Parity.